models/articles.models.js

```diff
@@ -31,6 +32,39 @@ function selectArticleById(article_id) {
     });
 }
 
+async function updateArticleById(article_id, inc_votes) {
+  if (inc_votes === undefined) {
+    return Promise.reject({ status: 400, msg: "Malformed Request" });
+  }
+  if (isNaN(article_id)) {
+    return Promise.reject({ status: 400, msg: "Id NaN" });
```

Comment:
This may not behave entirely as you expect as the article id comes through on the request params as a string